src/amo/pages/Feedback/index.js

```diff
@@ -0,0 +1,478 @@
+/* @flow */
```

Comment:
When we hit an API error, the button is still in a loading state and there is no way for the user to do anything.

Reply:
Do we have a standard way of dealing with this in frontend?  I looked at how `UserProfileEdit` handled errors (returns a `NotFound` page) and attempted to do the same here?

Reply:
Errors dispatch actions in most cases, and we can react to that in the reducer, e.g. by clearing the `loading` flag.

Reply:
I've attempted to handle the error as the user abuse report reducer/saga does it.